common/src/test/java/org/opensearch/ml/common/conversation/ConversationMetaTests.java

```diff
@@ -30,7 +30,7 @@ public class ConversationMetaTests {
     @Before
     public void setUp() {
         time = Instant.now();
-        conversationMeta = new ConversationMeta("test_id", time, time, "test_name", "admin");
+        conversationMeta = new ConversationMeta("test_id", time, time, "test_name", "admin", "testApp");
```

Comment:
Could you add a test case covering the situation when applicationType is null?

Reply:
sure, applicationType null test case added